modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/util/ObjectEntryTestUtil.java

```diff
@@ -30,9 +30,9 @@
  */
 public class ObjectEntryTestUtil {
 
-	public static ObjectEntry addObjectEntry(
+	public static <T extends Serializable> ObjectEntry addObjectEntry(
```

Comment:
I don't know if it is worth to make it parameterizable instead of just `Serializable objectFieldValue`